test/cluster/upsert/01-create-sources.td

```diff
@@ -62,3 +62,11 @@ $ kafka-ingest format=avro topic=dbzupsert key-format=avro key-schema=${keyschem
 id creature
 -----------
 1  lizard
+
+# Ingest a broken key/value pair
+$ kafka-ingest format=bytes topic=dbzupsert key-format=bytes
+broken-key:bar
+
+# Ingest a broken value with a good key
+$ kafka-ingest format=bytes topic=dbzupsert key-format=avro key-schema=${keyschema} publish=true
+{"id": 2}bar2
```

Comment:
why are we testing this in the cluster test? just to make sure we test that we persist error correctly?

Reply:
Yep, exactly. `test/cluster` is set up to allow us to destroy and create storaged hosts at arbitrary points, which is useful for testing that this stuff does what we expect.